workflow/automation/execution_scripts/auto_submit.py

```diff
@@ -187,15 +187,20 @@ def submit_script_to_scheduler(script_name, target_machine=None, **kwargs):
     elif proc_type == const.ProcessType.IM_plot.value:
         im_plot_template = "{script_location} {csv_path} {station_file_path} {output_xyz_dir} {srf_path} {model_params_path} {mgmt_db_loc} {run_name}"
 
+        #params.MODEL_PARAMS no longer is available.
+        model_params_path = os.path.join(os.path.dirname(params.vm_params),f"model_params{params.sufx}")
+
```

Comment:
Looking at a recent vm_params.yaml this params entry exists so we should revert this change.

Reply:
> > How does this commit change anything? It seems like the paths have just been shuffled around, but no effective change to the behaviour of the script
> 
> In the template, {{submit_command}} is substituted with something like:
> 
> ```
> time python $IMPATH/calculate_ims.py /scale_wlg_nobackup/filesets/nobackup/nesi00213/RunFolder/EndToEndTest/tests/tmp_20211209_011034/Runs/2012p001403/2012p001403/BB/Acc/BB.bin b -o /scale_wlg_nobackup/filesets/nobackup/nesi00213/RunFolder/EndToEndTest/tests/tmp_20211209_011034/Runs/2012p001403/2012p001403/IM_calc/ -np 40 -i 2012p001403 -r 2012p001403 -t s    -a ATC_Steel_SCBF_3Story SAC_Steel_MF_3Story --OpenSees /nesi/project/nesi00213/opt/maui/software/OpenSees/3.2.0-CrayGNU-19.04/OpenSees
> end_time=`date +$runtime_fmt`
> ```
> 
> As the previous $IMPATH doesn't have calculate_ims.py, the slurm script generated from this template didn't run.

That makes sense, looks like it wouldn't have worked with current master.